cmd/controller-manager/app/controller_manager.go

```diff
@@ -143,6 +143,8 @@ func Run(controllerManagerOptions *options.ControllerManagerServer) error {
 		// TODO: disambiguate API errors
 		return fmt.Errorf("failed to get Service Catalog client configuration: %v", err)
 	}
+	// TODO (nilebox): Dirty temporary hack. Should whether accept as flag for application, or handle TLS correctly
```

Comment:
@duglin it is removed in the next commit: https://github.com/kubernetes-incubator/service-catalog/pull/944/commits/213f946a5f1dcb8e7a86c69d4d583fc054672313